### ISSUE TITLE
Repair UI to show materials required

### ISSUE THREAD

**Issue by Krwak** [cataclysmbnteam/Cataclysm-BN#1824]
### Is your feature request related to a problem? Please describe.

When repairing/reinforcing items/clothing you see materials required only after the act in messages.
I find this confusing and player unfriendly, especially when being low on said materials.

### Describe the solution you'd like

I'd like there being another line in the UI which would show what materials and their numbers to be be used in said repair.
![image](https://user-images.githubusercontent.com/72284934/188030897-f281c812-0c77-4ccc-a181-c9722bd537ad.png)
Something along the lines of:
Materials required: 4 leather patches or 4 kevlar plates

### Describe alternatives you've considered

none

**Comment by Drugitechnik** [comment]
Duplicate of #689
